js/views/endScreen.js

```diff
@@ -50,6 +50,13 @@ var EndScreen = createReactClass({
       opacity: this.props.skinConfig.endScreen.replayIconStyle.opacity
     };
 
+    if (this.props.controller.state.cast.connected) {
+      actionIconStyle = Utils.extend(
+        {fontSize: '125px'},
+        actionIconStyle
+      );
```

Comment:
Maybe in future it would be better to use "spread" operator instead of the function from Utils.js ? :-)

Reply:
Sure, I will update that to use the spread operator :)